### PR TITLE
Beta Initial release

### PR DESCRIPTION
**Pull request by deepaksibm**
Features:

    Zonal file shares with replica

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,195 @@
 # vpc-beta-go-sdk
```
Comment (astha-jain) [inline README.md]: This title can be removed

File: common/headers.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+	"runtime"
+)
+
+const (
+	HEADER_NAME_USER_AGENT = "User-Agent"
+
+	SDK_NAME = "vpc-beta-go-sdk"
+)
+
+// GetSdkHeaders - returns the set of SDK-specific headers to be included in an outgoing request.
```
Comment (astha-jain) [inline common/headers.go:14]: can these comments be cleaned up

Comment (deepaksibm) [reply common/headers.go:14]: this is the explanation of GetSdkHeaders which has parameters and return parameters. This is present in vpc-go-sdk as well. Don't we need this ?

File: README.md (hunk 1 above)
Comment (deepaksibm) [reply README.md]: removed

File: README.md [hunk 3]
```diff
@@ -1,2 +1,194 @@
-# vpc-beta-go-sdk
 Go client library to interact with the various IBM Cloud VPC Services APIs in Beta.
```
Comment (astha-jain) [inline README.md]: This line can be removed

File: README.md [hunk 4]
```diff
@@ -1,2 +1,198 @@
...
+
+### Note:
```
Comment (astha-jain) [inline README.md]: Note can go after line 13

File: README.md [hunk 5]
```diff
@@ -1,2 +1,199 @@
...
+![GitHub go.mod Go version](https://img.shields.io/github/go-mod/go-version/IBM/vpc-beta-go-sdk)
+[![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
+
+# IBM Cloud VPC Beta Go SDK Version 0.1.0
+Go client library to interact with the various [IBM Cloud VPC Services APIs](https://cloud.ibm.com/apidocs?category=vpc).
+
+This SDK uses [Semantic Versioning](https://semver.org), and as such there may be backward-incompatible changes for any new `0.y.z` version.
+
+### Note:
+ This SDK surfaces IBM Cloud VPC features with Beta maturity which are subject to change before general availability.
```
Comment (astha-jain) [inline README.md:14]: This SDK surfaces IBM Cloud VPC features with Beta maturity which are subject to change before general availability.  IBM Cloud VPC features with general availability can be found here

**Review by astha-jain — APPROVED** [review]
Looks good

**Pull request merged**